dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -90,3 +90,5 @@ jwkEndpointUrl.desc=Specifies a JWK end point URL.
 useSystemPropertiesForHttpClientConnections=Use system properties for HTTP client connections
 useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the JWT Consumer creates HTTP client connections. Set this property to true if you want the connections to use the http* and javax* system properties.
 
+elapsedNBF=Elapsed Not Before Claim
+elapsedNBF.desc=Identifies the elapsed time until JWT will start to be accepted for processing. The value must be a NumericDate. 
```

Comment:
A nit:  Change "will start" to "starts"

Reply:
Got it, I updated the description to reflect that change.